dialogue-clients/src/main/java/com/palantir/dialogue/clients/ReloadingClientFactory.java

```diff
@@ -531,6 +534,13 @@ private ImmutableList<TargetUri> getTargetUris(
                 }
             }
         }
+        if (targetUris.isEmpty() && failedToParse) {
```

Comment:
do we need to handle a case where e.g. `uris` contains a mix of both valid and invalid URIs? With this change, we will drop the invalid ones completely.

Reply:
In _most_ cases, an invalid uri won't be able to handle requests, so we want to avoid the possibility of sending data there. When it does work, conversion to URI is likely to fail to list proxies anyhow, so behavior may not be correct :-/